<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>933</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RoboTerminal</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="Editor" name="editor">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>20</y>
      <width>920</width>
      <height>231</height>
     </rect>
    </property>
    <property name="documentTitle">
     <string>Editor</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>933</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="fileToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
  </widget>
  <widget class="QToolBar" name="actionToolBar">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="actionSend"/>
   <addaction name="actionCheckSyntax"/>
   <addaction name="actionSettings"/>
  </widget>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="style.qrc">
     <normaloff>:/rc/settings.png</normaloff>:/rc/settings.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="toolTip">
    <string>Serial connection Settings</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="style.qrc">
     <normaloff>:/rc/disconnected.png</normaloff>:/rc/disconnected.png</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionSend">
   <property name="icon">
    <iconset resource="style.qrc">
     <normaloff>:/rc/run.png</normaloff>:/rc/run.png</iconset>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
   <property name="toolTip">
    <string>Sends edited file</string>
   </property>
  </action>
  <action name="actionCheckSyntax">
   <property name="icon">
    <iconset resource="style.qrc">
     <normaloff>:/rc/checkoff.png</normaloff>:/rc/checkoff.png</iconset>
   </property>
   <property name="text">
    <string>CheckSyntax</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="style.qrc">
     <normaloff>:/rc/newFile.png</normaloff>:/rc/newFile.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="style.qrc">
     <normaloff>:/rc/save.png</normaloff>:/rc/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="style.qrc">
     <normaloff>:/rc/open.png</normaloff>:/rc/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Editor</class>
   <extends>QTextEdit</extends>
   <header>editor.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="style.qrc"/>
 </resources>
 <connections/>
</ui>
